addon/templates/components/course-materials.hbs

```diff
@@ -1,117 +1,174 @@
-{{! template-lint-disable attribute-indentation }}
 <div class="material-list">
-  <h3>{{t "general.courseLearningMaterials"}} ({{get (await course.learningMaterials) "length"}})</h3>
-  {{#if (gt (get (await course.learningMaterials) "length") 0)}}
-    <table>
+  <h3>
+    {{t "general.courseLearningMaterials"}} ({{get (await this.course.learningMaterials) "length"}})
+  </h3>
+
+  {{#if (gt (get (await this.course.learningMaterials) "length") 0)}}
+    <span class="filter-course-lms">
+      <input
+        placeholder={{t "general.filterPlaceholder"}}
+        value={{this.clmQuery}}
+        oninput={{perform this.setClmQuery value="target.value"}}>
+    </span>
+
+    <table class="table__course">
```

Comment:
This class seems to only be used in tests. If it's for test selection you should use https://github.com/simplabs/ember-test-selectors which would look like `<table data-test-course-table>` .